File: Assets/Scripts/Dtos/GameMoves/TradeOfferDto.cs
using System;
using Assets.Scripts.GameMode.Trading.Models;

namespace Assets.Scripts.Dtos.GameMoves
{
    [Serializable]
    public class TradeOfferDto
    {
        public string fromUser;
        public string toUser;
        public ResourceGroup offered;
        public ResourceGroup requested;
    }
}

File: src/main/java/com/catan/catanbackend/model/dto/move_dtos/TradeOfferDto.java
package com.catan.catanbackend.model.dto.move_dtos;

import com.catan.catanbackend.model.ResourceGroup;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TradeOfferDto {
    private String fromUser;
    private String toUser;
    private ResourceGroup offered;
    private ResourceGroup requested;
}